source/text/fontloader.ts

```diff
@@ -1,12 +1,15 @@
 
-import { log, logIf, LogLevel } from './auxiliaries';
+import { log, logIf, LogLevel, assert } from '../auxiliaries';
```

Comment:
[tslint] Named imports must be alphabetized.

`assert` can be removed, as it is not used here anymore.

Reply:
@cgcostume assert is still unused.